contracts/src/token/link_token.cairo

```diff
@@ -102,7 +103,7 @@ mod LinkToken {
         }
```

Comment:
There's a TODO to expose the minter method a couple lines above, can we take care of that here too?